.eslintrc.js

```diff
@@ -40,5 +41,29 @@ module.exports = {
         document: "readonly",
       },
     },
+    {
+      files: ["**/cli.js"],
+      rules: {
+        "node/shebang": 0,
```

Comment:
This can be removed if https://github.com/mdn/yari/pull/2156 lands, as it just complains if it isn't a registered bin value